pymatgen/entries/compatibility.py

```diff
@@ -931,7 +923,7 @@ def get_adjustments(self, entry: AnyComputedEntry) -> list[EnergyAdjustment]:
         Raises:
             CompatibilityError if the entry is not compatible
         """
-        if entry.parameters.get("run_type") not in ("GGA", "GGA+U"):
+        if entry.parameters.get("run_type") not in {"GGA", "PBE", "GGA+U", "PBE+U"}:
```

Comment:
This complete block (plus a comment about only correcting GGA/GGA+U entries) can be  moved to `get_adjustments`